### PR TITLE
cert name revision

### PR DESCRIPTION
**Pull request by rj-adams**
ensure it matches cert name in namespace specification

### CODE REVIEW

**Review by psoleckimoj — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ The easiest way to run the app is to use docker compose to create the service an
 The app requires: 
 * hmpps-auth - for authentication
 * redis - session store and token caching
-
+npm i
```
Comment (tpmcgowan) [inline README.md:60]: was this change intended 🤔?

Comment (rj-adams) [reply README.md:60]: no probably finger trouble - could you amend it back in your next PR?

Comment (tpmcgowan) [reply README.md:60]: No problem - will do!
